compiler/damlc/daml-stdlib-src/DA/Internal/Prelude.daml

```diff
@@ -438,7 +452,7 @@ lookup  key ((x,y)::xys)
 enumerate : (Enum a, Bounded a) => [a]
 enumerate = [minBound .. maxBound]
 
--- | `zip` takes two lists and returns a list of corresponding pairs.
+-- | `zip` takes two lists and returns a list of corresponding tuples.
```

Comment:
I think pairs is better than tuples as it is more precise. Same in many places below.

Reply:
Ok, I can change back